### PR TITLE
chore(ci) : update dependencies

### PR DESCRIPTION
**Pull request by mmorel-35**
This PR updates dependencies and setup dependabot

### CODE REVIEW

**Comment by Jeeppler** [comment]
@mmorel-35  thank you.

@de-jcup and @sven-dmlr please have a look.

**Comment by Jeeppler** [comment]
@mmorel-35 unfortunately, the core maintainer of the plugin is on vacation until September. Once he is back, he will have a look at this pull request.

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -9,11 +9,12 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2.3.4
```
Comment (mmorel-35) [inline .github/workflows/gradle.yml:12]:
```suggestion
    - uses: actions/checkout@v2
```

File: .github/workflows/gradle.yml [hunk 2]
```diff
@@ -9,11 +9,12 @@ jobs:
...
     - name: Set up JDK 1.8
-      uses: actions/setup-java@v1
+      uses: actions/setup-java@v2.1.0
```
Comment (mmorel-35) [inline .github/workflows/gradle.yml:14]:
```suggestion
      uses: actions/setup-java@v2
```

**Comment by de-jcup** [comment]
@mmorel-35 - thanx for the contribution! I appreciate the usage of dependabot inside the plugins - at the eclipse plugin the dependabot PRs must be handled carefully, because only merging the PR from dependabot will not automatically update the libraries used inside the plugin, but this is a follow up issue - the usage of dependabot is a good joice.

I created https://github.com/Daimler/sechub-plugin-eclipse/issues/14 because we do not apply PRs without issues. Inside the issue I will also explain the follow up issue to have dependabot PRs automatically synchronized between gradle and eclipse "world".

**Review by de-jcup — APPROVED** [review]
@mmorel-35 : Thank you. All fine.

**Pull request merged**